ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlResponseHandlers.java

```diff
@@ -73,8 +73,8 @@ private static boolean isCommandStatusEntity(final JsonObject ksqlEntity) {
   }
 
   private static boolean isIfNotExistsWarning(final JsonObject ksqlEntity) {
-    return ksqlEntity.getString("statementText") != null
-        && ksqlEntity.getString("statementText").contains("IF NOT EXISTS")
+    return ksqlEntity.getString("message") != null
+        && ksqlEntity.getString("message").startsWith("Cannot add")
```

Comment:
I don't love this, but this is necessary until we start adding more information to warning entities.